website/docs/r/is_vpn_server_route.html.markdown

```diff
@@ -32,10 +32,9 @@ Review the argument reference that you can specify for your resource.
   When processing this property, check for and log unknown values. Optionally halt processing and surface the error, or bypass the VPN route on which the unexpected property value was encountered.
```

Comment:
translate, deliver, drop are not nested schema elements. there are available options